plumgrid_playbooks/roles/create_ifcs/files/helpers.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Copyright (c) 2016, PLUMgrid Inc, http://plumgrid.com
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# (c) 2016, Javeria Khan <javeriak@plumgrid.com>
+
+# This script passes variable value for PLUMgrid Data Directory Path.
+
+plumgrid_data_dir="/var/lib/plumgrid/plumgrid-data"
```

Comment:
these paths will change automatically depending on the what package is being installed.